app/views/shared/_navbar.html.erb

```diff
@@ -1,44 +1,66 @@
-<div class="navbar navbar-toggleable-md navbar-light">
-  <div class="background"></div>
-  <button class="navbar-toggler navbar-toggler-right collapsed" type="button" data-toggle="collapse" data-target="#navbar-supported-content" aria-controls="navbar-supported-content" aria-expanded="false" aria-label="toggle-navigation">
-    <span class="navbar-toggler-icon open"></span>
-    <span class="close" aria-hidden="true">&times;</span>
-  </button>
-  <div class="navbar-brand">
-    <%= render 'shared/logo' %>
+<!-- Modal Content -->
+<div id="navbar-supported-content" class="modal modal-fullscreen-menu fade" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <div class="navbar-brand">
+          <%= render 'shared/brand' %>
+        </div>
+        <button aria-label="Close" type="button" class="close" data-dismiss="modal">
+          <span class="sr-only">Close navigation</span>
+          <span>X</span>
+        </button>
+      </div>
+      <ul class="modal-links">
+        <% if user_signed_in? %>
+          <%= render 'shared/navbar_components/dashboard' %>
```

Comment:
Partials for every link is bit much IMO. There isn't enough code in each link to warrant its own partial. Can we put each link in here instead of in a partial please?